src/App.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setMovie] = useState('');
+
+  const visibleMovies = moviesFromServer
+    .filter(movies => movies.title.toLowerCase().includes(query.toLowerCase())
+    || movies.description.toLowerCase().includes(query.toLowerCase()));
```

Comment:
```suggestion
    || includeCheck(movies, query);
```

you can create a function to avoid duplicating your code)